src/containers/sidebars/data-global-sidebar/analyze-areas-sidebar-card/component.jsx

```diff
@@ -128,13 +132,20 @@ function AnalyzeAreasCardComponent({
                 <RadioButton
                   id={option.slug}
                   option={{ ...option, name: option.label }}
-                  checked={selectedOption.slug === option.slug}
+                  checked={selectedOption?.slug === option.slug}
                   onChange={() => handleOptionSelection(option)}
                   theme={radioTheme}
                 />
               </div>
             );
           })}
+          <button
+            type="button"
+            className={styles.clearFilters}
+            onClick={clearFilters}
+          >
+            clear filters
```

Comment:
We think that 'Clear selection' is more accurate as a text. As these are more options for the user to interact than filters. Although they do filter and select layers